### PR TITLE
Cassandra yml configuration

### PR DESCRIPTION
**Pull request by AbhijithGanesh**
Addresses #2

### CODE REVIEW

**Comment by AbhijithGanesh** [comment]
The commit log is because of upstream merges, apologies.

File: src/config/cassandra.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Cassandra storage config YAML
+cluster_name: "Single Node Cluster"
```
Comment (patchandpray) [inline src/config/cassandra.yml:2]: why are you choosing a name with spaces here, wouldn't camelCase or snake_case be more appropriate?

File: src/config/readme.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+The cassandra yml configures how the local Cassandra instance needs to be configured.
```
Comment (shreemaan-abhishek) [inline src/config/readme.md:1]: What do you think about replacing this line with, "The cassandra.yml file configures the local Cassandra instance."

File: src/config/readme.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+The cassandra yml configures how the local Cassandra instance needs to be configured.
+It uses the configuration coomands in `cassandra.yml` to define provisions. For most provisions the comment lines
```
Comment (shreemaan-abhishek) [inline src/config/readme.md:2]: There's a typo in commands 😂 

Also, what is a provision?

File: src/config/readme.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+The cassandra yml configures how the local Cassandra instance needs to be configured.
+It uses the configuration coomands in `cassandra.yml` to define provisions. For most provisions the comment lines
+elaborate the configured provision.
+
+The [provisions](https://cassandra.apache.org/doc/latest/cassandra/configuration/cass_yaml_file.html) can be read here. 
+
+The schema needs to be manually configured after the start-up is ready. To do that, open the shell attached to the container
```
Comment (shreemaan-abhishek) [inline src/config/readme.md:7]: What is a startup? Or how do I start it up? Another question, what do I need to start-up?

File: src/config/readme.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 The cassandra yml configures how the local Cassandra instance needs to be configured.
-It uses the configuration coomands in `cassandra.yml` to define provisions. For most provisions the comment lines
+It uses the configuration comands in `cassandra.yml` to define provisions. For most provisions the comment lines
```
Comment (shreemaan-abhishek) [inline src/config/readme.md:2]:
```suggestion
It uses the configuration commands in `cassandra.yml` to define provisions. For most provisions the comment lines

```

**Review by AmitKarnam — APPROVED** [review]
LGTM, Most of the intended changes have already been requested.

**Review by AmitKarnam — COMMENTED** [review]
Looks good to go.

File: src/config/readme.md (hunk 4 above)
Comment (shreemaan-abhishek) [reply src/config/readme.md:8]: @AbhijithGanesh

**Pull request closed without merging**